cdap-docs/_common/_source/index.rst

```diff
@@ -106,6 +106,12 @@ on the installation, monitoring and diagnosing fully distributed CDAP in a Hadoo
 - |faq|_ Answers to questions you might have about CDAP
 
 
+.. |search| replace:: **Search:**
+.. _search: search.html
+
+- |search|_ Search this documentation, using either *Quick Search* or *Google*
```

Comment:
I see. So both Quick Search and Search with Google are using Google Search? Even after your explanation though, I'm still confused. Just by looking at the page, I wouldn't know what button to use to search.